httpie/__main__.py

```diff
@@ -12,6 +12,9 @@ def main():
     except KeyboardInterrupt:
         from . import ExitStatus
         sys.exit(ExitStatus.ERROR_CTRL_C)
+    except ModuleNotFoundError:
+        from core import main
```

Comment:
I believe this worked for you, but it doesn't make sense and there's not enough detail in this hybrid bug report/pull request to reproduce the error you're seeing.

Please give us far more detailed instructions to reproduce this on our own machines. This includes operating system, version of python, version of pyinstaller, version of httpie dependencies, etc.

Reply:
This was done with Python 3.6.4, Windows 7, HTTPie 0.9.9 and PyInstaller 3.3.1.